Docs/ProjectSpec.md

```diff
@@ -130,6 +132,84 @@ options:
+    - `SwiftErrorBreakpoint`: swift error breakpoint
+    - `OpenGLErrorBreakpoint`: OpenGL breakpoint
+    - `SymbolicBreakpoint`: symbolic breakpoint
+    - `IDEConstraintErrorBreakpoint`: IDE constraint breakpoint
+    - `IDETestFailureBreakpoint`: IDE test failure breakpoint
+- [ ] **enabled**: **Bool** - Indicates whether it should be active. Default to `true`
+- [ ] **ignoreCount**: **Int** - Indicates how many times it should be ignored before stopping, Default to `0`
+- [ ] **continueAfterRunningActions**: **Bool** - Indicates if should automatically continue after evaluating actions, Default to `false`
+- [ ] **timestamp**: **String** - Breakpoint timestamp
+- [ ] **filePath**: **String** - Breakpoint file path (only required by file breakpoints)
+- [ ] **line**: **Int** - Breakpoint line (only required by file breakpoints)
+- [ ] **breakpointStackSelectionBehavior**: **String** - Breakpoint stack selection behavior
+- [ ] **symbol**: **String** - Breakpoint symbol (only used by symbolic breakpoints)
+- [ ] **module**: **String** - Breakpoint module (only used by symbolic breakpoints)
+- [ ] **scope**: **String** - Breakpoint scope (only used by exception breakpoints)
+    - `All`
+    - `Objective-C` (default)
+    - `C++`
+- [ ] **stopOnStyle**: **String** - Indicates if should stop on style (only used by exception breakpoints)
+    -`throw` (default)
+    -`catch`
+- [ ] **condition**: **String** - Breakpoint condition
+- [ ] **actions**: **[[Breakpoint Action](#breakpoint-action)]** - breakpoint actions
+
+```yaml
+breakpoints:
+  - type: ExceptionBreakpoint
+    enabled: true
+    ignoreCount: 0
+    continueAfterRunningActions: false
+```
+
+#### Breakpoint Action
+
+- [x] **type**: **String** - Breakpoint action type
+    - `DebuggerCommand`: execute debugger command
+    - `Log`: log message
+    - `ShellCommand`: execute shell command
+    - `GraphicsTrace`: capture GPU frame
+    - `AppleScript`: execute AppleScript
+    - `Sound`: play sound
+    - `OpenGLError`: throw OpenGL error
+- [ ] **consoleCommand**: **String** - Debugger command (only used by debugger command breakpoint action)
+- [ ] **message**: **String** - Log message (only used log message breakpoint action)
+- [ ] **conveyanceType**: **String** - Conveyance type (only used by log message breakpoint action)
+    - `console`: log message to console (default)
+    - `speak`: speak message
+- [ ] **command**: **String** - Shell command file path (only used by shell command breakpoint action)
+- [ ] **arguments**: **String** - Shell command arguments (only used by shell command breakpoint action)
+- [ ] **waitUntilDone**: **Bool** - Indicates whether it should wait until done (only used by shell command breakpoint action). Default to `false`
+- [ ] **script**: **String** - AppleScript (only used by AppleScript breakpoint action)
+- [ ] **soundName**: **String** - Sound name (only used by sound breakpoint action)
+    - `Basso` (default)
+    - `Blow`
```

Comment:
Can we lowercase these

Reply:
We can if we restore them before setting them to `XcodeProj `.

Reply:
Can I ask the reason?
There are some possible values that are not lowercase, so I am confused.
Are we going to make possible values lowercase from now on?

Reply:
You're right. These don't need to follow swift lower camel casing, and other enums here use uppercase.